xclim/indices/run_length.py

```diff
@@ -67,18 +69,23 @@ def rle(
     da : xr.DataArray
     dim : str
     max_chunk : int
+    lastday: bool
 
     Returns
     -------
     xr.DataArray
-      Values are 0 where da is False (out of runs),
-      are N on the first day of a run, where N is the length of that run,
+      Values are 0 where da is False (out of runs).
+      If lastday is False, values are N on the first day of a run, where N is the length of that run,
+      and are NaN on the other days of the runs.
+      If lastday is True, values are N on the last day of a run, where N is the length of that run,
       and are NaN on the other days of the runs.
     """
     use_dask = isinstance(da.data, dsk.Array)
 
     # Ensure boolean
     da = da.astype(bool)
+    if lastday:
+        da = da.reindex(time=da.time[::-1])
```

Comment:
As noted below, but flagging it here, we should avoid hardcoding a dimension name here!

Reply:
Done :)